### PR TITLE
[PL-1106] - Prospective storyscripter can see single example on the playground welcome page

### PR DESCRIPTION
**Pull request by Arinono**
As a prospective storyscripter,
When I navigate to the playground welcome page,
I see a single example giving me context,
So that I know what I’m going to be looking at when I load it

## Acceptance

**When** I navigate to the playground welcome page
**Then** I see a single card with the redis logo and the title counter.story with the textual description:

```
A simple web page hit counter application, showing the power of Storyscript for rapid prototyping using the http and redis services.
```

**And When** I click the Explore button
**Then** I am navigated to the playground page with that example loaded

## Notes
Design (minus the FAQ part)
playground-w

### CODE REVIEW

File: src/assets/img/redis.svg [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <defs>
+            <clipPath id="clip0">
+              <rect
+                width="70.875"
+                height="60.75"
+                fill="white"
+              />
+            </clipPath>
+          </defs>
+        </svg>
```
Comment (matthewhudson) [inline src/assets/img/redis.svg]: Can you remove the extra tabs/spaces? Alternatively, and FWIW, we can save some bytes by running this through a formatter like [svgomg](https://jakearchibald.github.io/svgomg/).

File: src/assets/styles/custom/_scrollbar.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
+ * Container style
+ */
+
+ .ps {
```
Comment (matthewhudson) [inline src/assets/styles/custom/_scrollbar.scss]: Minor nit: I think these lines may have an extra leading space.

File: src/assets/img/redis.svg (hunk 1 above)
Comment (Arinono) [reply src/assets/img/redis.svg]: Done 👍

File: src/assets/styles/custom/_scrollbar.scss (hunk 2 above)
Comment (Arinono) [reply src/assets/styles/custom/_scrollbar.scss]: Done 👍

**Comment by Arinono** [comment]
@JeanBarriere Ready for your review. I did not add a Percy screenshot for the intro as it will probably change in the next few days.

File: src/router.ts [hunk 3]
```diff
@@ -8,12 +8,33 @@ export default new Router({
   base: process.env.BASE_URL,
   routes: [
     {
-      path: '/',
-      name: 'playground',
-      component: () => import('@/views/Playground/index.vue')
+      path: '',
+      component: () => import('@/Playground.vue'),
```
Comment (JeanBarriere) [inline src/router.ts]: remove this one

Comment (Arinono) [reply src/router.ts]: Done 👍

**Pull request merged**